README.md

```diff
@@ -131,15 +131,16 @@ These instructions are tested and verified on macOS Catalina (10.15).
     You might encounter the following errors. The possible fixes are also provided herewith.
      * Error installing gem: ***mimemagic***
      * Fix-> `brew install shared-mime-info`
-     {{< /note >}}
+     * If you get error installing **therubyracer** and **libv8** on macos, follow the steps outlined in this link: [https://billykong.github.io/ruby/2020/03/17/fixing-libv8-in-osx-catalina.html](https://billykong.github.io/ruby/2020/03/17/fixing-libv8-in-osx-catalina.html)
```

Comment:
Nit: Can we change macos to camel-case **macOS**?

Reply:
Sure

Reply:
Done. Approve if all looks good @RoyShravani